modules/ROOT/pages/about-designing-a-mule-application.adoc

```diff
@@ -4,10 +4,50 @@ include::_attributes.adoc[]
 endif::[]
 :keywords:
 
-In Flow Designer, you create Mule applications as projects having one or more flows. A project contains the flows for one Mule application. A Mule application can have one or more flows. A flow consists of a sequence of _cards_, with each card representing a connector, core Mule Runtime component, API, or module. Each card receives an input, carries out a specific task, and then sends the output of that task to the next card in the flow. The final card in a flow is usually one that sends a notification that the Mule application has completed its task or that sends data.
+:imagesdir: ../assets/images
 
-To get started with Flow Designer by building an example Mule application, see xref:salesforce-to-twilio.adoc[Create Your First Mule Application with Flow Designer].
+In Flow Designer, you create Mule applications as projects having one or more flows. A project contains the flows for one Mule application. A Mule application can have one or more flows. A flow consists of a sequence of _cards_, with each card representing a connector, core Mule Runtime component, API, or module. Each card receives an input, carries out a specific task, and then sends the output of that task to the next card in the flow. The final card in a flow is usually one that sends a notification that the Mule application has completed its task or that sends data.
 
+== Getting Started
+
+To learn about Flow Designer by building an example Mule application, you can try the tutorial xref:salesforce-to-twilio.adoc[Create Your First Mule Application with Flow Designer]. The tutorial guides you in creating a Mule app that places bulk orders for company swag automatically when opportunities are closed, and that notifies customers via SMS about their order status and ETA.
+
+To start creating Mule applications:
+
+* Try the easiest option, which is to import a template into Flow Designer from Anypoint Exchange. See xref:import-templat.adoc[Create a Mule Application Quickly from a Template].
+* Create a new Mule-app project in Design Center and use the wizard that automatically opens to start a project with a trigger card and a target card.
```

Comment:
do we need "Mule-app" here?  Can we delete it and just say new project?  I think it would be easier for users  who don't know what Mule apps are.  Same comment on the next line.

Reply:
When a user clicks **Create** in Design Center, he's given a choice of creating a Mule app or importing a project. So, I think we need to keep "Mule-app" here.

Reply:
Also, the procedure begins with "To start creating Mule applications:". And both terms are used in the paragraph that precedes the procedure. Therefore, "Mule-app" does have context.